ntbs-service/wwwroot/source/Components/ValidateMultiple.ts

```diff
@@ -0,0 +1,56 @@
+import Vue from 'vue';

```

Comment:
I tried to make this component general, so it can be used whenever we need to validate multiple components at once. We might be able to reuse/extend this so it also works for e.g. contact tracing validation, but I haven't looked into that yet.